drivers/filesystems/ext2/CMakeLists.txt

```diff
@@ -86,11 +86,13 @@ list(APPEND SOURCE
 
 add_library(ext2fs SHARED ${SOURCE} ext2fs.rc)
 
+if(USE_CLANG_CL)
+    set_property(SOURCE src/create.c src/fileinfo.c src/memory.c src/read.c APPEND_STRING PROPERTY COMPILE_FLAGS " /fallback")
```

Comment:
In theory this is a nice flag, but it means we cannot use clang on linux?

Reply:
clang-cl, not clang :)
Please note that 32bit SEH support is not hooked up yet for anything other than MSVC, so only clang-cl has it.